pmd-core/etc/xslt/pmd-report-v2.xslt

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
```

Comment:
Since we produce html, it's probably better to explicitly configure html output:

```xml
<xsl:output method="html"/>
```

Otherwise the default might be xml.